### ISSUE TITLE
[BUG] Pull requests are failing 😢

### ISSUE THREAD

**Issue by gunwant11** [DesignSystemsOSS/thedesignsystems#42]
### 🙁 Actual behavior

idk why pull request of many people including me are failing 
is there an issue in depolyment ??

### 🙂 Expected behavior

Pull request should be successful 😤

**Comment by pulkithanda** [comment]
@gunwant11 Are you talking about the merge conflicts?

**Comment by Yash-Chauhan09** [comment]
@gunwant11 I think pull requests are not failing, it's just that maybe the owner of the repo can only be able to manage the deployment with vercel.

**Comment by yashsehgal** [comment]
Yes, @gunwant11 The vercel team account has ended and that's why the documentation deployment is failing. We are working on setting up a new domain and host the site there.
